javascript.md

```diff
@@ -1363,6 +1365,32 @@ the asterisk should be sticked to the `yield` keyword:
 
 [&#8593; back to TOC](#table-of-contents)
 
+## ECMAScript 2017
+
+### async-await
+
+* Don't use `return await`
+
+  > Explanation: Since the return value of an async function is always wrapped in `Promise.resolve`,
+  > return await doesn’t actually do anything except add extra time before the overarching `Promise`
```

Comment:
s/overarching/returned/ ?

Reply:
I just copy-and-paste description from [eslint](https://eslint.org/docs/rules/no-return-await)